src/_zkapauthorizer/tests/test_client_resource.py

```diff
@@ -493,7 +493,8 @@ def test_post(self, get_config, api_auth_token, voucher, unblinded_tokens):
             dumps(
                 {
                     u"unblinded-tokens": list(
-                        token.unblinded_token for token in unblinded_tokens
+                        token.unblinded_token.encode("ascii")
```

Comment:
`unblinded_tokens` is a list of `UnblindedToken` and `UnblindedToken.unblinded_token` is now `bytes` instead of `unicode` so this `encode` is unnecessary (though "safe" because it's a base64 string).  Right?

Reply:
This should be *decode*, since json (as a format, not necessarily the implementation) doesn't support byte strings.